### PR TITLE
Only track experiments that are set in the config

### PR DESCRIPTION
**Pull request by lucidlogic**

### CODE REVIEW

File: tests/Unit/Helpers/ExperimentsTest.php [hunk 1]
```diff
@@ -40,14 +41,42 @@ public function testExperimentGroup()
     public function testTrackExperiments()
     {
         $experiments = [
-            'experiment-1' => 'test',
+            'recommend' => 'test',
         ];
 
         $_COOKIE['experiments'] = json_encode($experiments);
 
-        GoogleTagManager::shouldReceive('set')
```
Comment (lucidlogic) [inline tests/Unit/Helpers/ExperimentsTest.php:48]: this doesn't work btw  😓

**Review by erhardt-ringiersa — APPROVED** [review]

**Review by rklugman — APPROVED** [review]

**Review by SheldonGrant — APPROVED** [review]
will this not lead to loss of leads/views, etc that we aren't tracking at the moment.
Like unattributed or other? Disclaimer: didn't actually read the code.

**Comment by lucidlogic** [comment]
"will this not lead to loss of leads/views" no, this is not stopping any events tracked, its just prevents the experiment group to be sent to GTM if the experiment is not set in the config

**Comment by rklugman** [comment]
Maybe if the users experiment cookie isn’t a running experiment, randomize to something that’s not running. In this case, only control is set so will always be that?

**Pull request merged**